proxy/backendtimeout_test.go

```diff
@@ -112,8 +123,9 @@ func TestBackendTimeoutInTheMiddleOfServiceResponse(t *testing.T) {
 		t.Errorf("expected partial content, got %s", content)
 	}
 
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Since we need to wait for the logs in all cases, lets make `containsEveryLineCount` signature the same as `WaitFor`, i.e. require duration argument.